### ISSUE TITLE
[1.6.0a10] CLI Unable to rename files

### ISSUE THREAD

**Issue by jaybarbee** [comictagger/comictagger#590]
I have had a couple of examples where I am able to pull data down and save the cr info in CBZ, but when I run a command to rename the file via CLI.   If I use the GUI, the rename works fine.

Command I am using:
`comictagger.exe --rename -t cr 'comic.cbz'`

Using Windows "ComicTagger-1.6.0a10-win-AMD64.exe.zip" version with my script.

**Comment by lordwelch** [comment]
Please provide the logs at '%APPDATA%\ComicTagger\logs'

**Comment by jaybarbee** [comment]
My ComicTagger.log file in that location hasn't been written to since 10/15/2023.   I am not seeing an option in --help to allow debug logging.   Is there something I can enable?  Also while looking for a debug flag to enable, I see you have --template.  Do I need to specify a template?

**Comment by lordwelch** [comment]
That's odd. You can also add `-vv` and that will give you all of the logs.
It will also start with lines like this telling you where it is logging to:
```plain
2024-01-20T10:29:37 | comictagger | INFO | ComicTagger Version: 1.6.0a11.dev3 running on: Darwin PyInstaller: No
2024-01-20T10:29:37 | comictagger | DEBUG | user_data_dir: /Users/timmy/Library/Application Support/ComicTagger
2024-01-20T10:29:37 | comictagger | DEBUG | user_config_dir: /Users/timmy/Library/Application Support/ComicTagger
2024-01-20T10:29:37 | comictagger | DEBUG | user_cache_dir: /Users/timmy/Library/Caches/ComicTagger
2024-01-20T10:29:37 | comictagger | DEBUG | user_state_dir: /Users/timmy/Library/Application Support/ComicTagger
2024-01-20T10:29:37 | comictagger | DEBUG | user_log_dir: /Users/timmy/Library/Logs/ComicTagger
```

**Comment by jaybarbee** [comment]
Ah... I see.   It is in "%UserProfile%\AppData\Local\ComicTagger\ComicTagger\Logs"   If you use %AppData% that puts you into AppData\Roaming...

Let me run a quick rename test and I will post it.

**Comment by jaybarbee** [comment]
Here is the command I used:
`comictagger.exe --rename -t cr '.\Wolverine 041 (2024) (digital) (Marika-Empire).cbz' -vv`

Here is what is in my CommicTagger.log

```plain
2024-01-20T13:34:01 | comictagger | INFO | ComicTagger Version: 1.6.0a10 running on: Windows PyInstaller: Yes
2024-01-20T13:34:01 | comictagger | DEBUG | user_data_dir: C:\Users\jay\AppData\Local\ComicTagger\ComicTagger
2024-01-20T13:34:01 | comictagger | DEBUG | user_config_dir: C:\Users\jay\AppData\Local\ComicTagger\ComicTagger
2024-01-20T13:34:01 | comictagger | DEBUG | user_cache_dir: C:\Users\jay\AppData\Local\ComicTagger\ComicTagger\Cache
2024-01-20T13:34:01 | comictagger | DEBUG | user_state_dir: C:\Users\jay\AppData\Local\ComicTagger\ComicTagger
2024-01-20T13:34:01 | comictagger | DEBUG | user_log_dir: C:\Users\jay\AppData\Local\ComicTagger\ComicTagger\Logs
2024-01-20T13:34:01 | comictagger | DEBUG | Installed Packages
2024-01-20T13:34:01 | comictagger | DEBUG | comicinfoxml	0.1.2
2024-01-20T13:34:01 | comictagger | DEBUG | comictagger	1.6.0a10
2024-01-20T13:34:01 | comictagger | DEBUG | importlib-metadata	7.0.1
2024-01-20T13:34:01 | comictagger | DEBUG | metron_talker	0.1.4
2024-01-20T13:34:01 | comictaggerlib.cli | INFO | 
2024-01-20T13:34:01 | comictaggerlib.cli | INFO | --------- Comic Rack tags ---------
2024-01-20T13:34:01 | comictaggerlib.cli | INFO | series:      Wolverine
issue:       41
issue_count: 41
title:       Sabretooth War: Part 1
publisher:   Marvel
year:        2024
month:       3
day:         1
web_link:    https://comicvine.gamespot.com/wolverine-41-sabretooth-war-part-1/4000-1039590/
story_arcs:  ['Fall of X', 'Sabretooth War']
characters:  Northstar, Black Tom Cassidy, Sage, Quentin Quire, Aurora, Sabretooth, Daken, Domino, Wolverine
teams:       X-Force
locations:   North Pole, The Greenhouse, The Grove, Krakoa, Arctic
description: SABRETOOTH WAR BEGINS HERE!

THE MOST VIOLENT WOLVERINE STORY EVER TOLD!

Get ready for the showdown to end all showdowns - WOLVERINE VS. SABRETOOTH!

It's been years since these heavy hitters have crossed paths in the Marvel Universe, but as Krakoa falls, so rises SABRETOOTH - and he's out for revenge! They threw VICTOR CREED in the PIT, but he's free and, wielding an army of Sabretooths, will prove once and for all why he is LOGAN's ultimate nemesis.

Co-written by literary powerhouses Benjamin Percy (WOLVERINE, GHOST RIDER) and Victor LaValle (The Changeling, SABRETOOTH) and drawn by artistic dynamos Cory Smith (CONAN, GHOST RIDER) and Geoff Shaw (WEAPONS OF VENGEANCE, WOLVERINE: PATCH), don't miss the inaugural issue in this multi-shipping saga leading up to the landmark issue #50!
notes:       Tagged with ComicTagger 1.6.0a10 using info from Comic Vine on 2024-01-19 22:16:02. [Issue ID 1039590]
credit:      Writer: Benjamin Percy
credit:      Writer: Victor LaValle
credit:      Penciller: Cory Smith
credit:      Penciller: Geoff Shaw
credit:      Inker: Geoff Shaw
credit:      Inker: Oren Junior
credit:      Colorist: Alex Sinclair
credit:      Letterer: Cory Petit
credit:      Cover: Clayton Crain
credit:      Cover: Greg Capullo
credit:      Cover: Guru-eFX
credit:      Cover: Leinil Francis Yu
credit:      Cover: Mark Bagley
credit:      Cover: Romulo Fajardo Jr.
credit:      Cover: Salvador Larroca
credit:      Editor: C.B. Cebulski
credit:      Editor: Drew Baumgartner
credit:      Editor: Jordan D. White
credit:      Editor: Mark Basso
```

This pretty much matches the output of what I see in Windows Terminal, but if you want that output too, I can provide as well.

**Comment by lordwelch** [comment]
So turns out I accidentally made `-r, --rename` the same as `-p, --print`. That will be fixed later today

**Comment by lordwelch** [comment]
This was fixed in bb1a83b. It will be in the next release
